### ISSUE TITLE
Feature:  Loading skeleton in Hero 

### ISSUE THREAD

**Issue by NsdHSO** [open-sauced/hot#297]
### Type of feature

🍕 Feature

### Current behavior

If you search in the search box after one repo, we receive responses 1-2 seconds. I think we should display a skeleton 

### Suggested solution

```
     if (isLoading) {
    return (
      <div className="p-4 border rounded-2xl bg-white w-full space-y-1 relative">
       // Hint Array.from(Array(5)) || props count = {1,2,3....}
        <Skeleton
          enableAnimation
          count={1}
        />
      </div>
    );
  }
```

### Additional context

issue:
fix #297 

PR:
#296 

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

**Comment by bdougie** [comment]
not technically a good first issue since it is blocked by another PR.

**Comment by SunGoldTech** [comment]
.take

**Comment by SunGoldTech** [comment]
I'm thinking about making the hero card a separate reusable component, to stay organized... @bdougie  what do u think about that?

**Comment by NsdHSO** [comment]
Yes, good remarks, but you don't have this to do with this issue. See here, we get the answer after 1-2 seconds. 
![image](https://user-images.githubusercontent.com/37635083/184304952-f9657365-4859-4d13-8632-cfd0d0cc1de8.png)

**Comment by bdougie** [comment]
Closing this for now. however, we do need a skeleton for RepoWrap instead. I will open that issue.
